doc/sphinx/practical-tools/utilities.rst

```diff
@@ -163,8 +169,8 @@ Then:
 
 - Compile your project with :n:`make -f CoqMakefile` as needed.
 
-If you add new files to your project, add them to `_CoqProject` and
-re-run `coq_makefile` and `make`.
+If you add more files to your project that are not in directories listed
```

Comment:
Is that correct? Since `coq_makefile` is responsible for traversing the directories, I don't understand how a new file could be located without running it again.

Reply:
I mistakenly dropped the expansion of directories in the makefile, I'll re-add that.